go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -234,7 +237,7 @@ func (c *addressManager) updateNodeAddressAnnotations() error {
 	}
 
 	// update k8s.ovn.org/host-addresses
-	if err = util.SetNodeHostAddresses(c.nodeAnnotator, c.addresses); err != nil {
+	if c.updateHostAddresses(node) != nil {
```

Comment:
heh, need err to be set here:
`if err = c.updateHostAddresses(node); err != nil`
?
If not, I would like this to explicitly check for the err value that makes it okay to be returning `nil`, as well as returning that explicitly: `return nil` after a TODO header explaining why.

Reply:
Aahh this was an oversight on my part, thanks Flavio. Updated

Reply:
np